packages/typechain-target-web3-v1/lib/index.ts

```diff
@@ -44,40 +44,28 @@ export default class Web3 extends TsGeneratorPlugin {
       {
         path: join(this.outDirAbs, "types.d.ts"),
         contents: `
-  import { EventLog } from "web3/types";
-  import { TransactionObject, BlockType } from "web3/eth/types";
   import BN from "bn.js";
+  import { EventLog } from "web3/types";
+  import { BlockType } from "web3/eth/types";
   import { EventEmitter } from "events";
-  // @ts-ignore
```

Comment:
can u explain these changes?

Reply:
@krzkaczor - There's a `import { TransactionObject } from "web3/eth/types"` that can be leveraged which works in the same way as the types defined above. Also, instead of `EstimateGasOptions`, I think there's a type of `Tx` which provides all the available options for `.call()`, `.send()` etc. Thanks!

Reply:
I think it would be better to do this in a separate PR, this is potentially a breaking change because I believe that `TransactionObject` was added in recent web3v1 versions. L

ets try to keep this PR focused on adding v2 target - changes normalizing naming etc are fine.

Reply:
Sure, I'll remove those changes then 👍